autoload/tataku.vim

```diff
@@ -10,30 +10,49 @@ function! tataku#call_recipe(recipe_name) abort
   call denops#notify('tataku', 'run', [l:recipe])
 endfunction
 
-let s:recipe_name = ''
-function! tataku#_setup_operator(recipe_name) abort
+" operator{{{
+function! tataku#_operator(recipe_name) abort
   let s:recipe_name = a:recipe_name
-endfunction
 
-function! tataku#_call_as_operator(motion_type) abort
-  if s:is_invalid_region()
-    return
-  endif
+  function! s:operator(...) abort
```

Comment:
If length of arguments will not be over than 2,
using named argument is preffer to extra arguments.